pkg/client/util_test.go

```diff
@@ -260,15 +263,16 @@ func TestReconcilePostgres(t *testing.T) {
 		{
 			name: "test modification function error",
 			args: args{
-				ctx:            context.TODO(),
-				client:         fake.NewFakeClientWithScheme(scheme),
-				deploymentType: "workshop",
-				tier:           "development",
-				productName:    "test",
-				name:           "test",
-				ns:             "test",
-				secretName:     "test",
-				secretNs:       "test",
+				ctx:              context.TODO(),
+				client:           fake.NewFakeClientWithScheme(scheme),
+				deploymentType:   "workshop",
+				tier:             "development",
+				productName:      "test",
+				name:             "test",
+				ns:               "test",
+				secretName:       "test",
+				secretNs:         "test",
+				applyImmediately: false,
```

Comment:
could we add a case here which tests applyImmediately is true.

Reply:
sure

Reply:
actually I will modify the 
```
			name: "test modification function",
```